examples/app-settings/package.json

```diff
@@ -20,6 +20,7 @@
     "devDependencies": {
         "@eslint/js": "^9.9.0",
         "@frontify/frontify-cli": "^5.7.7",
+        "@types/node": "^20.12.12",
```

Comment:
Is this really needed? All our code run in the browser, no?

Reply:
yeah, otherwise there is an issue with console.log in the editor. Its for typescript to properly detect "node".

Reply:
Isn't the underlying problem that we have two different tsconfig files? 

What I tried in`sdk-apps-code-examples/examples/app-with-state`. 
- `rm tsconfig.json`
- `mv tsconfig.app.json tsconfig.json`
- `npm run lint` -> no error
- `npm run typecheck` no error

Why do we need two configs?

Reply:
Shouldnt be an issue as we have an import in the tsconfig.json
```
{
    "files": [],
    "references": [
        { "path": "./tsconfig.app.json" },
        { "path": "./tsconfig.node.json" }
    ]
}
```

Reply:
That is probably because it has no issue before, no?

Reply:
Though, what is `./tsconfig.node.json` for?

Reply:
the node file is used for the environmental files, like eslint.config.js, tailwind.config.ts, postcss.config.js files. It doesnt include browser specific but node specific configurations.

Reply:
But it still needed the "jsx": "react" thing, which is a bit weird, 🤔

Reply:
The problem is that by default all files are included: https://www.typescriptlang.org/tsconfig/#include

So, `tsconfig.node.json` should actually have a `"exclude": ["src"]`.

Reply:
Ill try this one, might be possible to then remove the jsx:react in the node one. Pretty cumbersome...